src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -75,11 +75,11 @@ const computeSubProcessKind = (processedSemanticType: BpmnSemanticType, bpmnElem
   }
 };
 
-// specific management for adhoc and transaction sub-processes which are handled with a dedicated ShapeBpmnSubProcessKind
-// process boundary events afterward as we need its parent activity to be available when building it
 const orderedFlowNodeBpmnTypes: BpmnSemanticType[] = [
   'adHocSubProcess',
   'transaction',
+  // specific management for adhoc and transaction sub-processes which are handled with a dedicated ShapeBpmnSubProcessKind
```

Comment:
**suggestion:** move this line above the first "sub-process" values
Currently, the comment is written after these values which make things hard to understand IMHO.